core/src/Threads/Kokkos_Threads_Team.hpp

```diff
@@ -675,15 +675,15 @@ class TeamPolicyInternal<Kokkos::Threads, Properties...>
                      const Kokkos::AUTO_t& /* team_size_request */
                      ,
                      const Kokkos::AUTO_t& /* vector_length_request */)
-      : TeamPolicyInternal(space, league_size_request, -1, -1) {}
+      : TeamPolicyInternal(space, league_size_request, 1, -1) {}
 
   /** \brief  Specify league size, request team size*/
   TeamPolicyInternal(const typename traits::execution_space& space,
                      int league_size_request,
                      const Kokkos::AUTO_t& /* team_size_request */
                      ,
                      int vector_length_request)
-      : TeamPolicyInternal(space, league_size_request, -1,
+      : TeamPolicyInternal(space, league_size_request, 1,
```

Comment:
Passing `1` might not be the best choice, but at least this should get CI going :crossed_fingers: 
For comparison, `OpenMP` goes with `m_space.impl_thread_pool_size(2)`, which is a fancy way to pass `1` as well.
https://github.com/kokkos/kokkos/blob/6fc7a4930ff106bc720b15b526ac63aede05c7a0/core/src/OpenMP/Kokkos_OpenMP_Team.hpp#L232-L242
https://github.com/kokkos/kokkos/blob/6fc7a4930ff106bc720b15b526ac63aede05c7a0/core/src/OpenMP/Kokkos_OpenMP.hpp#L169-L171